app/api-requests/application-api.config.schema.js

```diff
@@ -1,7 +1,5 @@
-const Joi = require('joi')
+import joi from 'joi'
 
-const schema = Joi.object({
-  uri: Joi.string().uri().default('http://host.docker.internal:3001/api')
+export const schema = joi.object({
```

Comment:
probably worth calling this applicationApiConfigSchema, depending on how it is used